### DISCUSSION TITLE
How to disable httpyac requests used as tests in vscode

### DISCUSSION THREAD

**Discussion by aaksarin** [AnWeber/vscode-httpyac#307] [Q&A] [answered]
Hi. Is there a way to disable auto adding httpyac requests as tests in vscode?

I don't use http requests as tests and don't want them to run if I run all tests in project.

**Comment by AnWeber** [comment, marked as answer]
Disable TestExplorer support with settings: [httpyac.testControllerEnabled](https://github.com/AnWeber/vscode-httpyac/blob/main/package.json#L991)

**Reply by aaksarin** [reply]
Thank you
